src/screens/Authentication/PhoneNumber/PhoneNumber.tsx

```diff
@@ -9,37 +9,44 @@ import {
 } from 'react-hook-form';
 import { FirebaseRecaptchaVerifierModal } from 'expo-firebase-recaptcha';
 import PhoneInput from 'react-native-phone-number-input';
+import { AuthStackScreenProps } from '../../../types/navigation';
 import styles from './styles';
 import {
   phoneGetConfirmation,
   getActivationStatus,
 } from '../../../firebase/auth';
 import { firebaseApp } from '../../../firebase/config';
 import StyledButton from '../../../components/StyledButton/StyledButton';
-import logo from '../../../assets/cnsc-logo.png';
-import WelcomeScreen from '../Welcome/Welcome';
-import { AuthStackScreenProps } from '../../../types/navigation';
+
+const logo = require('../../../assets/cnsc-logo.png');
 
 function PhoneNumberScreen({
   navigation,
 }: AuthStackScreenProps<'PhoneNumberScreen'>) {
   interface FormValues {
     phoneNumber: string;
   }
+
   const { ...methods } = useForm();
   const recaptchaVerifier = useRef(null);
   const [phoneNumber, setPhoneNumber] = useState('');
   const [valid, setValid] = useState(false);
```

Comment:
not sure if we need a useState for this valid variable. i think we can get away with just checking the output of the validatePhoneNumber function and setting that to a variable